### ISSUE TITLE
unused type, false positive

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1478]
```abap
CLASS lcl_class DEFINITION.
  PUBLIC SECTION.
    TYPES ty_bar TYPE string.
    METHODS m RETURNING VALUE(asdf) TYPE lcl_class=>ty_bar.
ENDCLASS.
CLASS lcl_class IMPLEMENTATION.
  METHOD m.
  ENDMETHOD.
ENDCLASS.
```

![image](https://user-images.githubusercontent.com/5888506/96970394-eb3ae700-1513-11eb-9bc1-121d4f801f1a.png)

**Comment by larshp** [comment]
done, closing
